packages/mdc-ripple/_mixins.scss

```diff
@@ -156,6 +186,22 @@
   }
   // stylelint-enable at-rule-empty-line-before, block-closing-brace-newline-after
 
+  &.mdc-ripple-upgraded {
+    &#{$pseudo} {
+      top: 0;
+      left: 0;
+      width: var(--mdc-ripple-fg-size);
```

Comment:
Yeah, if we're checking in JS, there's no need for the first fallback. Still might be useful to do `var(--mdc-ripple-fg-size, 16px)` (or whatever sensible default), in case `--mdc-ripple-fg-size` isn't defined.